cflib/crazyflie/param.py

```diff
@@ -313,6 +358,166 @@ def get_value(self, complete_name, timeout=60):
         [group, name] = complete_name.split('.')
         return self.values[group][name]
 
+    def persistent_clear(self, complete_name, callback=None):
+        """
+        Clear the current value of the specified persistent parameter from
+        eeprom. The supplied callback will be called with `True` as an
+        argument on success and with `False` as an argument on failure.
+
+        @param complete_name The 'group.name' name of the parameter to store
+        @param callback Optional callback should take boolean status as arg
+        """
+        element = self.toc.get_element_by_complete_name(complete_name)
+
+        def new_packet_cb(pk):
+            if pk.channel == MISC_CHANNEL and pk.data[0] == MISC_PERSISTENT_CLEAR:
+                callback(pk.data[3] == 0)
```

Comment:
`callback` is optional and possibly None, I can not see any check that it actually contains a function before it is called. Maybe I miss something but otherwise a `not None` check should be added somewhere.

Reply:
Yes, nice catch! Thanks!

Reply:
I added a check for callback and forced push. Also make sure to remove port callbacks now.